packages/plugin-search/src/search.js

```diff
@@ -0,0 +1,115 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {pick, isArray} from 'lodash';
+import {SparkPlugin} from '@ciscospark/spark-core';
+
+const Search = SparkPlugin.extend({
+  namespace: `Search`,
+
+  people(options) {
+    options = options || {};
+
+    if (!options.query && !options.queryString) {
+      return Promise.reject(new Error(`'options.query' is required`));
+    }
+
+    const body = pick(options, `size`, `includePeople`, `includeRooms`, `includeRobots`);
```

Comment:
that said, you might want to default all of those values to false, thus forcing people to opt into each response type